build/automation/var/profile/test.mk

```diff
@@ -16,90 +16,8 @@ DB_PORT := 5432
 DB_NAME := pathwaysdos_regressiondi
 DB_SCHEMA := pathwaysdos
 DB_SECRET_NAME := core-dos-dev/deployment
-DB_SECRET_KEY := DB_SF_READONLY_PASSWORD
+DB_SECRET_KEY := DB_DI_READONLY_PASSWORD
 DB_USER_NAME_SECRET_NAME = uec-dos-int-dev/deployment
 DB_USER_NAME_SECRET_KEY = DOS_REPLICA_DI_USERNAME
-TF_VAR_dos_db_name := $(DB_SERVER_NAME)
 
-# ==============================================================================
-# Infrastructure variables (Terraform, Serverless, etc)
-
-# Change Event Receiver API Gateway API Keys
-TF_VAR_api_gateway_api_key_name := $(PROJECT_ID)-$(ENVIRONMENT)-api-key
-TF_VAR_nhs_uk_api_key_key := NHS_UK_API_KEY
-
-# Lambda Security Group
-TF_VAR_lambda_security_group_name := $(PROJECT_ID)-$(ENVIRONMENT)-lambda-sg
-
-# Change Event Receiver API Gateway Route53 & SQS
-TF_VAR_dos_integration_sub_domain_name := $(PROGRAMME)-$(TEAM_ID)-$(ENVIRONMENT)
-DOS_INTEGRATION_URL := $(TF_VAR_dos_integration_sub_domain_name).$(TEXAS_HOSTED_ZONE)/v1/change-event
-TF_VAR_di_endpoint_api_gateway_name := $(PROJECT_ID)-$(ENVIRONMENT)-di-endpoint
-TF_VAR_di_endpoint_api_gateway_stage := $(ENVIRONMENT)
-TF_VAR_fifo_queue_name := $(PROJECT_ID)-$(ENVIRONMENT)-fifo-queue.fifo
-TF_VAR_dead_letter_queue_from_fifo_queue_name := $(PROJECT_ID)-$(ENVIRONMENT)-dead-letter-queue.fifo
-SQS_QUEUE_URL:= https://sqs.$(AWS_REGION).amazonaws.com/$(AWS_ACCOUNT_ID)/$(TF_VAR_fifo_queue_name)
```

Comment:
The `SQS_QUEUE_URL` is still being used on some make targets for testing

Reply:
good spot

Reply:
i put it back on the end